#!/bin/bash
sudo python3 recover.py
